ui/src/containers/CreateVolume.js

```diff
@@ -513,12 +633,108 @@ const CreateVolume = (props) => {
                         name="path"
                         value={values.path}
                         onChange={handleChange('path')}
-                        label={intl.translate('device_path')}
+                        label={
+                          <>
+                            {intl.translate('device_path')}
+                            <Tooltip
+                              placement="right"
+                              overlay={
+                                <div style={{ minWidth: '200px' }}>
+                                  {intl.translate('device_path_explanation')}
+                                </div>
+                              }
+                            >
+                              <InputQuestionMark className="fas fa-question-circle"></InputQuestionMark>
+                            </Tooltip>
+                          </>
+                        }
                         error={touched.path && errors.path}
                         onBlur={handleOnBlur}
                       />
                     )}
+                    <CheckboxContainer>
+                      <Checkbox
+                        name="multiVolumeCreation"
+                        label={intl.translate('create_multiple_volumes')}
+                        checked={values.multiVolumeCreation}
+                        value={values.multiVolumeCreation}
+                        onChange={handleChange('multiVolumeCreation')}
+                        onBlur={handleOnBlur}
+                      />
+                    </CheckboxContainer>
                   </FormSection>
+
+                  {values.multiVolumeCreation && (
+                    <MultiCreationFormContainer>
+                      <FieldArray
+                        name="volumes"
+                        render={(arrayHelpers) => (
+                          <div>
+                            <div
+                              style={{
+                                paddingLeft: '60px',
+                                paddingTop: `${padding.large}`,
+                              }}
+                            >
+                              <span
+                                style={{
+                                  paddingRight: `${padding.base}`,
+                                }}
+                              >
+                                {intl.translate('number_volume_create')}
+                              </span>
+                              {/* TODO: Generalize the number input component to core-ui. */}
+                              <InputNumberComponentStyle
```

Comment:
I think we should handle a max number of volumes to avoid dom freeze in case of the user mistakenly ask 10000 volumes, simply add a max prop here should be enough

Reply:
Indeed, good point!
I am going to set the `max` number to`70` for the moment.